include/eve/conditional.hpp

```diff
@@ -315,14 +361,28 @@ namespace eve
     static constexpr bool is_inverted     = false;
     static constexpr bool is_complete     = false;
 
-    constexpr keep_between_(std::ptrdiff_t b, std::ptrdiff_t e) noexcept : begin_(b), end_(e) {}
+    constexpr keep_between_(std::ptrdiff_t b, std::ptrdiff_t e) noexcept
+    : begin_(std::min(b,e)), end_(std::max(b,e))
```

Comment:
Why do you need min here?
These couple of instructions can be an issue when the whole thing takes 5 ns

Reply:
I wanted to support the case where begin is greater than end.
I may just assert if this error occurs?